packages/generator/package.json

```diff
@@ -18,16 +18,19 @@
   ],
   "scripts": {
     "depcheck": "depcheck",
+    "lint": "eslint . --ext .ts --quiet --fix",
     "pretest": "npm run lint && npm run depcheck",
-    "test:debug": "nyc mocha \"test/**/*.test.ts\"",
+
+    "build": "npm run renderTemplates && tsc && eslint dist --ext .ts,.js --quiet --fix",
     "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports/generator.xml",
-    "test:integration": "nyc --check-coverage=false mocha \"test_integration/**/*.ts\" --retries 2 --timeout 5000 --reporter=xunit --reporter-options output=../../reports/generator-integration.xml",
```

Comment:
Is there a reason removing test:integration?

Reply:
Yes, integration tests should now all live in the integration test folder.

Reply:
These tests are not part of integrationTests folder yet. Until it is moved, we should keep testing those

Reply:
For some reason I was thinking your story moved those.

Reply:
Talking with @monkey-jeff we'll leave these disabled for now as they don't provide huge value and re-address the tests in W-7487432.